hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -0,0 +1,639 @@
+          desc: '''
+            When the OPERATION field is programmed to generate HW output, this field selects
+            the appropriate crypto cipher target.
+          ''',
+          resval: "0"
+          enum: [
+            { value: "0",
+              name: "None",
+              desc: '''
+                No target selected
+                '''
+            },
+            { value: "1",
+              name: "AES",
+              desc: '''
+                AES selected
+                '''
+            },
+            { value: "2",
+              name: "HMAC",
+              desc: '''
+                HMAC selected
+                '''
+            },
+            { value: "3",
+              name: "KMAC",
+              desc: '''
+                KMAC selected
+                '''
+            },
+          ]
+        },
+      ],
+    },
+
+    { name: "ROM_EXT_DESC_EN",
+      desc: "Register write enable for ROM_EXT_DESC",
+      swaccess: "rw0c",
+      hwaccess: "none",
+      fields: [
+        { bits: "1:0",
+          name: "EN",
+          resval: "2"
+          desc: '''
+            ROM_ext descriptor configure enable.
+            All values except for 2 are interpreted as configuration disable.
+          '''
+        },
+      ]
+    },
+    { multireg: {
+        cname: "KEYMGR",
+        name: "ROM_EXT_DESC",
+        desc: '''
+          ROM_ext descriptor.
+          This value must be representative of the current ROM_ext contents.
+          For example, this could be a truncated hash of the ROM_ext
+        ''',
+        count: "NumRomExtReg",
+        swaccess: "rw",
+        hwaccess: "hro",
+        fields: [
+            { bits: "31:0",
+              name: "VAL",
+              desc: '''
+                Software binding value
+              ''',
+              resval: "0"
+            },
+        ],
+      },
+    },
+
+    // The following can potentially be replaced by a RAM if it gets too large
+    // Currently there are 32b * NumInReg * 3 bits for inputs
+    //                     32b * NumOutReg * 2 (if key shares desired) for outputs
+    //                     32b * NumOutReg * NumCryptos * 2 (if key shares desired) for hidden key bus
+    // This totals to over 2kb of storage
+
+    { multireg: {
+        cname: "KEYMGR",
+        name: "SOFTWARE_BINDING",
+        regwen: "CFGEN",
+        desc: '''
+          Software binding input to key manager.
+          This register is non-lockable and shared between various software stages.
+          This binding value is not considered secret.
+        ''',
+        count: "NumInReg",
+        swaccess: "rw",
+        hwaccess: "hro",
+        fields: [
+            { bits: "31:0",
+              name: "VAL",
+              desc: '''
+                Software binding value
+              ''',
+              resval: "0"
+            },
+        ],
+      },
+    },
+
+    { multireg: {
+        cname: "KEYMGR",
+        name: "KEY_ID",
```

Comment:
@moidx 
just wanted to check-in on your thoughts for this currently? As currently defined, both salt and key ID are still present.

Reply:
I am leaning towards combining these two inputs since construction wise they will probably end up next to each other. 

How about we remove it for now?

Reply:
yeah we can do that. Since i'm working to get this in for bronze, i might keep it for now and file an issue. 

Feel free to add more to that issue on things to combine and remove, how does that sound?

Reply:
SGTM

Reply:
i've removed the key_id!